HW0/P5/P5.txt

```diff
@@ -1 +1,54 @@
 # Your answers here.
+
+1. It would take 255 or 256 seconds (depending on whether or not you count
+adding the first value to 0 as a second or not) 
+
```

Comment:
The answer to this is 255 seconds. Since it would make more sense to add the first two numbers directly rather than add it to zero.